portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteApps.testcase

```diff
@@ -286,6 +286,47 @@ definition {
 		}
 	}
 
+	@description = "Verify that remote app of type Custom Element can save multiple URLs"
+	@priority = "4"
+	test CustomElementCanSaveMultipleURLs {
+		property portal.acceptance = "true";
+
+		var customElementName = "Vanilla Counter";
+		var customElementHTMLName = "vanilla-counter";
+		var customElementURL1 = "http://remote-component-test.wincent.com/packages/vanilla-counter/index1.js";
+		var customElementURL2 = "http://remote-component-test.wincent.com/packages/vanilla-counter/index2.js";
+		var customElementCSSURL1 = "http://remote-component-test.wincent.com/index1.css";
+		var customElementCSSURL2 = "http://remote-component-test.wincent.com/index2.css";
```

Comment:
Let's remove these lines and just name the variable from the macro call. I want the variables just named in the macro as much as possible to make them more readable for debugging/test analysis.

Also, good job on picking up the custom Element URLs from other tests. We need to update these though since we are in middle of migrating the URL host. Please replace with links from https://issues.liferay.com/browse/LPS-142823
ie: 
https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js
https://liferay.github.io/liferay-frontend-projects/index.css